test/nntrainer_test_util.cpp

```diff
@@ -243,3 +244,37 @@ bool getMiniBatch_val(float *outVec, float *outLabel, int *status) {
   F.close();
   return true;
 }
+
+/**
+ * @brief return a tensor filled with contant value with dimension
+ */
+nntrainer::Tensor constant(float value, unsigned int batch, unsigned channel,
+                           unsigned height, unsigned width) {
+  nntrainer::Tensor t(batch, channel, height, width);
+
+  float * dat = t.getData();
+
+  for(unsigned int i = 0; i < t.getDim().getDataLen(); ++i){
+    dat[i] = value;
+  }
```

Comment:
```suggestion
  t.setValue(value);
```

Reply:
Uhh right... I misunderstood that. Will fix right away. Thanks 👍